aspnetcore/host-and-deploy/visual-studio-publish-profiles.md

```diff
@@ -81,9 +57,23 @@ When MSBuild or Visual Studio loads a project, the following high-level actions
 
 ## Compute project items
 
-When the project is loaded, the project items (files) are computed. The `item type` attribute determines how the file is processed. By default, *.cs* files are included in the `Compile` item list. Files in the `Compile` item list are compiled.
+When the project is loaded, the project items (files) are computed. The item type determines how the file is processed. By default, *.cs* files are included in the `Compile` item list. Files in the `Compile` item list are compiled.
```

Comment:
Crosslink to https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-items?view=vs-2019?